sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -162,6 +162,12 @@ def __init__(self, packages, options, environment_version):
             value=to_json_value(job_type)),
         dataflow.Environment.VersionValue.AdditionalProperty(
             key='major', value=to_json_value(environment_version))])
+    if job_type.startswith('FNAPI_'):
+      runner_harness_override = (
+          dependency.get_runner_harness_container_image())
+      if runner_harness_override:
+        self.debug_options.expriments.append(
```

Comment:
<!--thread_id:cc_143607418_t; commit:0df823c8839461643514dc954b59c2e073253fb0; resolved:1-->
<!--section:context-quote-->
> **charlesccychen** wrote:
> "experiments"?

<!--section:body-->
Thanks for catching. Added a unit test.